### PR TITLE
Adding CICD pipeline dashboard

### PR DESCRIPTION
**Pull request by bblinder**
Creating a templatized version of our CICD pipeline dashboard

### CODE REVIEW

**Comment by bblinder** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by bblinder** [comment]
I have read the Code of Conduct and I hereby accept the Terms

**Comment by greatestusername-splunk** [comment]
Thank you for contributing!! 🚀 
Will give this a review once I've got a moment.

**Comment by greatestusername-splunk** [comment]
May be worth writing a `README.md` that describes what metrics this is using, along with the important dimensions used for various things and where to set them in variables.tf? Could make this a really useful template for all sorts of users.

**Comment by bblinder** [comment]
Thanks for the feedback! I've added a bunch of these and will make another pull request.

File: dashboards-and-dashboard-groups/CICD-Pipeline/CICD_autoremediation.tf [hunk 1]
```diff
@@ -0,0 +1,480 @@
...
+  auth_token = var.access_token
+}
+
+# signalfx_event_feed_chart.CICD_autoremediation_0:
+resource "signalfx_event_feed_chart" "CICD_autoremediation_0" {
+  name         = "Event Feed"
+  program_text = <<-EOF
+        C = events(eventType='Automated Rollback initiated').publish(label='C')
+        B = events(eventType='canary push event').publish(label='B')
+        A = alerts(detector_name='CI/CD Service Outlier').publish(label='A')
```
[the remark this answers is not in the document]
Comment (rcastley) [reply dashboards-and-dashboard-groups/CICD-Pipeline/CICD_autoremediation.tf:12]: The generator behind the CI/CD pipeline demo is not public, this is a public repo so just wondering how the public would use/benefit from this?

Internally, this is awesome just trying to understand the benefits for inclusion in a public repo.

**Pull request closed without merging**